c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +341,18 @@ app.DocumentEditingController.prototype.submitForm = function(isValid) {
 
 
 /**
+ * @param {appx.Document} data Document attributes.
+ * @return {appx.Document}
+ * @export
```

Comment:
Same question here: Why `export`? `@export` makes sure that a property/method is not renamed by the compiler so that you can call if from a template. But that doesn't seem to be the case?